Examples/Statistics/ImageHistogram4.cxx

```diff
@@ -156,7 +156,7 @@ main(int argc, char * argv[])
   // using the \code{SetHistogramBinMinimum()} and
   // \code{SetHistogramBinMaximum()} methods.
   //
-  // Software Guide : EndLatexex
```

Comment:
This probably caused some problems during software guide generation. Good to have it fixed!

Reply:
These doc fixes are awesome :100:. Thanks Dave.

I had a look to the SW Guide out of curiosity, and the block was being processed correctly; even the figure name fix in https://github.com/InsightSoftwareConsortium/ITK/pull/4008/files#diff-f001c5fd30fbde05e83c609f5cc7c143b5ef4f87c85ea91e25cabd0c32ef8df7R275 or a close typo in https://github.com/InsightSoftwareConsortium/ITK/pull/4008/files#diff-f001c5fd30fbde05e83c609f5cc7c143b5ef4f87c85ea91e25cabd0c32ef8df7R257 are not present in the [ITK SW Guide version in itk.org](https://itk.org/ItkSoftwareGuide.pdf) (April 16, 2021). Weird.